src/main/java/seedu/address/logic/commands/CompleteCommand.java

```diff
@@ -0,0 +1,139 @@
+package seedu.address.logic.commands;
+
+import static java.util.Objects.requireNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_APPOINTMENT_DATE;
+import static seedu.address.model.Model.PREDICATE_SHOW_ALL_PERSONS;
+
+import java.time.LocalDate;
+import java.util.List;
+import java.util.Objects;
+import java.util.Optional;
+import java.util.function.Consumer;
+
+import seedu.address.commons.core.index.Index;
+import seedu.address.commons.util.CollectionUtil;
+import seedu.address.logic.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.model.Model;
+import seedu.address.model.appointment.NullAppointment;
+import seedu.address.model.person.Person;
+
+/**
+ * Completes and removes appointments from Person.
+ */
+public class CompleteCommand extends Command {
+
+    public static final String COMMAND_WORD = "complete";
+
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": Completes an appointment identified by "
+            + "by the index number used in the displayed person list or "
+            + "by a specified date"
+            + "Parameters: INDEX(must be a positive integer) "
+            + "[" + PREFIX_APPOINTMENT_DATE + "Appointment Date] "
+            + "Example: " + COMMAND_WORD + " 1 ";
+    public static final String MESSAGE_COMPLETE_SUCCESS = "Appointments Completed!";
+    public static final String MESSAGE_NOT_COMPLETED = "At least one identifier must be provided.";
```

Comment:
can you change this to provide more specific details on how to complete the command.